server/src/dbUtils.ts

```diff
@@ -15,6 +17,17 @@ function getStringOrUndefined(value: unknown): string | undefined {
 	throw `Expected value to be string or null, got: ${typeof value}`;
 }
 
+function getTagsOrUndefined(value: unknown): string[] | undefined {
+	if (value === null) {
+		return undefined;
+	}
+
+	if (!isTags(value)) {
```

Comment:
And here maybe we could call isStringArray() to also keep this abstract. This way we would be able to reuse this for other stringArrays

Reply:
aah, didn't notice this comment. Ok, will do! :)